### ISSUE TITLE
Minimap without item

### ISSUE THREAD

**Issue by Ozzymud** [l3lessed/l3lessed-DFUnity-Mods#5]
Using https://github.com/l3lessed/l3lessed-DFUnity-Mods/blob/main/l3lessed%20Enchanted%20Compass%20-%20Minimap%20mod/Vortex%20Installer/l3lessed%20Enchanted%20Compass%20-%20Windows.rar that was posted on or about 5/9.

I had previously had the minimap.dfmod installed on my save but it had no colors on buildings and I didn't know about all the settings to disable the effects over the compass like snow and blood so I disabled the mod. Upon reloading my game I had some unknown items in my inventory (the old lens, repair kit, compass, etc) so I dropped them on the ground and continued on without the minimap.

Recently I was watching a video and saw the settings for the mod outside of the game. I then reinstalled the mod, loaded the game and adjusted the settings to remove all the environmental effects as well as de-selecting the option to need the compass equipped. Loaded the save and... nothing... double pressing the numpad + key changed my cursor to what it was normally in ingame settings but no minimap or settings scroll.

I then went hunting and found the thread on dfworkshop.net and saw that there was a new version (doesn't appear on nexus?). So I came here, grabbed it, and installed "Blessed Enchanted Compass.dfmod" instead of "minimap.dfmod". Loaded up the save and nothing still. Saw a users post over on nexus about him giving himself a compass by editing the save file, so I did that with the items from a new save... repair kit, gears, glass and the compass. Loaded the save and the items were back. Still no minimap without equipping the compass though.

So, TL;DR of all this is how do I get the minimap without equipping the compass?
